ourpath.py

```diff
@@ -0,0 +1,83 @@
+import osmnx as ox
+import networkx as nx
+from shapely.geometry import Polygon
+import matplotlib.pyplot as plt
+import folium
+
+    # Function to save plot as image
+def plot_graph(G):
+    fig, ax = ox.plot_graph(G, show=False, close=False)
+    return fig, ax
+    
+    
+def get_path(want_graph):
```

Comment:
looks good for now.
We'll need further modifications like using a predefined path (example - College Path) & then have another function to add nodes and edges to it. 
We also need the function to take parameters of orig_node , dest_node. (future implementation)